src/app/kiosk/kiosk.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { KioskRoutingModule } from './kiosk-routing.module';
```

Comment:
From now, we will separate Modules importation from Components importation with an empty line.